Clicker/ViewControllers/EditQuestionTypeViewController.swift

```diff
@@ -0,0 +1,124 @@
+//
+//  EditQuestionTypeViewController.swift
+//  Clicker
+//
+//  Created by Matthew Coufal on 9/18/18.
+//  Copyright © 2018 CornellAppDev. All rights reserved.
+//
+
+import UIKit
+
+protocol EditQuestionTypeDelegate {
+    func didPick(questionType: QuestionType)
+}
+
+class EditQuestionTypeViewController: UIViewController {
+
+    var delegate: EditQuestionTypeDelegate?
+    var selectedQuestionType: QuestionType!
+    
+    // MARK: - View vars
+    var multipleChoiceButton: UIButton!
+    var multipleChoiceLabel: UILabel!
+    var freeResponseButton: UIButton!
+    var freeResponseLabel: UILabel!
+    var selectedDot: UIView!
+    
+    // MARK: - Constants
+    let dotDiameter: CGFloat = 15
+    let dotPadding: CGFloat = 15
+    let labelRightPadding: CGFloat = 45
+    let labelLeftPadding: CGFloat = 15
+    let buttonHeight: CGFloat = 50
+    
+    init(selectedQuestionType: QuestionType) {
+        super.init(nibName: nil, bundle: nil)
+        self.selectedQuestionType = selectedQuestionType
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        view.backgroundColor = .white
+
+        multipleChoiceButton = UIButton()
+        multipleChoiceButton.addTarget(self, action: #selector(didPickMultipleChoice), for: .touchUpInside)
+        view.addSubview(multipleChoiceButton)
+        
+        multipleChoiceLabel = UILabel()
+        multipleChoiceLabel.text = "Multiple Choice"
+        multipleChoiceLabel.textColor = selectedQuestionType == .multipleChoice ? .aquaMarine : .clickerBlack0
+        multipleChoiceLabel.font = selectedQuestionType == .multipleChoice ? ._16SemiboldFont : ._16RegularFont
+        view.addSubview(multipleChoiceLabel)
+        
+        freeResponseButton = UIButton()
+        freeResponseButton.addTarget(self, action: #selector(didPickFreeResponse), for: .touchUpInside)
+        view.addSubview(freeResponseButton)
+        
+        freeResponseLabel = UILabel()
+        freeResponseLabel.text = "Free Response"
+        freeResponseLabel.textColor = selectedQuestionType == .freeResponse ? .aquaMarine : .clickerBlack0
+        freeResponseLabel.font = selectedQuestionType == .freeResponse ? ._16SemiboldFont : ._16RegularFont
+        view.addSubview(freeResponseLabel)
+        
+        selectedDot = UIView()
+        selectedDot.backgroundColor = .aquaMarine
+        selectedDot.clipsToBounds = true
+        selectedDot.layer.cornerRadius = dotDiameter / 2
+        view.addSubview(selectedDot)
+        
+        layoutSubviews()
+    }
+    
+    func layoutSubviews() {
+        
+        multipleChoiceButton.snp.makeConstraints { make in
```

Comment:
Instead of putting these constraints in `layoutSubviews`, can you just make a function called `setupConstraints` and put them in there? or do you need to have it in `layoutSubviews`?